tests/test_e2e_06_topology.py

```diff
@@ -104,38 +104,43 @@ def test_015_delete_link(self):
     def test_020_delete_switch(self):
         """Test api/kytos/topology/v3/switches/{switch_id} on DELETE
         Deleted:
-            - Links: Ampath3 - Ampath2; Ampath3 - Ampath1
-            - Switch: Ampath2
+            - Links: s1 - s2
+            - Switch: s1
         """
         # Switch is not disabled, 409
-        switch = "00:00:00:00:00:00:00:17"
-        api_url = f'{KYTOS_API}/topology/v3/switches/{switch}'
+        switch_1 = "00:00:00:00:00:00:00:01"
+        api_url = f'{KYTOS_API}/topology/v3/switches/{switch_1}'
         response = requests.delete(api_url)
         assert response.status_code == 409
 
         # Switch have links, 409
-        api_url = f'{KYTOS_API}/topology/v3/switches/{switch}/disable'
+        api_url = f'{KYTOS_API}/topology/v3/switches/{switch_1}/disable'
         response = requests.post(api_url)
         assert response.status_code == 201
 
-        api_url = f'{KYTOS_API}/topology/v3/switches/{switch}'
+        api_url = f'{KYTOS_API}/topology/v3/switches/{switch_1}'
         response = requests.delete(api_url)
         assert response.status_code == 409
 
         # Get the link_id
-        api_url = f'{KYTOS_API}/topology/v3/links'
-        response = requests.get(api_url)
-        assert response.status_code == 200
-        data = response.json()
+        counter_searches = 0
         links_id = list()
-        for key, value in data['links'].items():
-            if (value["endpoint_a"]["switch"] == switch or 
-                value["endpoint_b"]["switch"] == switch):
-                links_id.append(key)
-        assert links_id
-
-        self.net.net.configLinkStatus('Ampath3', 'Ampath2', 'down')
-        self.net.net.configLinkStatus('Ampath3', 'Ampath1', 'down')
+        # No longer assert link list, this run could be a retry.
+        while not links_id and counter_searches < 5:
```

Comment:
Acceptable to roll out the retry here since it's a simple one. But, let's keep in mind (for next PRs when adding retries) to also use `tenacity` as we do for our NApps, `tenacity` also supports retry with its [contexts manager](https://tenacity.readthedocs.io/en/latest/#retrying-code-block), which can be handy when we also don't want to extract a method to be decorated:

```
 from tenacity import Retrying, RetryError, stop_after_attempt

   try:
       for attempt in Retrying(stop=stop_after_attempt(3)):
           with attempt:
               raise Exception('My code is failing!')
   except RetryError:
       pass
```

Reply:
Got it, will do for future PRs